### ISSUE TITLE
Migrating from v3 to v4, instructions reference terrariumpi.db but doesn't exist?

### ISSUE THREAD

**Issue by Benjin** [theyosh/TerrariumPI#627]
I'm following the instructions [here](https://theyosh.github.io/TerrariumPI/install/#migrate) to migrate from v3 to v4 in order to test the fix for #231, but I'm not spotting the referenced terrariumpi.db.  I've renamed the old directory, cloned the new installation, and run `install.sh`.  Where should this be located?

**Comment by alec-pinson** [comment]
try

`find / -name "*terrariumpi.db*" 2>/dev/null`

**Comment by theyosh** [comment]
It could be that the documentation is not that clear. The file `terrariumpi.db` should be in the same folder as the code. But it will only be there, when TP4 is started at least once. Then the db is created. So after running the new `install.sh` you should reboot once. That should trigger the start of TP4, which will create the new database.

Also, with 4.1, with docker support, will change the database location, so I think I have to address that as well in the documentation.

**Comment by theyosh** [comment]
And you have to configure the relays and sensors in TP4 first, before a migration can take place. So make sure TP4 is configured and running. Then shutdown TP4 and do the migration step.

**Comment by Benjin** [comment]
I see; yeah, I think the documentation could be improved a bit?  Things that might be worth calling out explicitly:

* After TP3 is backed up and renamed, do a normal, complete installation of TP4 (to the point that the web UI is live)
* Relays/sensors must be configured _before_ the migration
  * Why can't this part of the migration step?  `copy_relay_settings.py` clearly has the core info: name, type of relay, address.
  * What is it migrating if not the name/type/address; just history?  Since the script name includes "settings", I figured those would be included since they're the main settings.

In any case, I've got it running now.  Feel free to close this issue, or perhaps leave it open to track documentation updates?
